pledges/forms.py

```diff
@@ -17,15 +17,15 @@ def clean(self):
 
         # 1. The URL should match the domain of the site
         site = self.cleaned_data['site']
-        domain_re = '([\w\.]+\.)?{}'.format(site.domain.replace('.', '\.'))
+        domain_re = r'([\w\.]+\.)?{}'.format(site.domain.replace('.', r'\.'))
         parsed_url = urlparse(self.cleaned_data['url'])
         if not re.match(domain_re, parsed_url.netloc):
             raise ValidationError(
                 'URL domain must match or be a subdomain of the site domain.'
             )
 
         # 2. The contact email address should match the domain of the site
-        email_re = '.+@{}'.format(site.domain.replace('.', '\.'))
```

Comment:
wow i don't even think we use this part of the site

Reply:
You're right, we don't! See https://github.com/freedomofpress/securethenews/pull/163 for snipping it out.